rust/agama-network/src/nm/dbus.rs

```diff
@@ -1302,7 +1328,10 @@ mod test {
     use crate::{
         model::*,
         nm::{
-            dbus::{BOND_KEY, ETHERNET_KEY, INFINIBAND_KEY, WIRELESS_KEY, WIRELESS_SECURITY_KEY},
+            dbus::{
+                BOND_KEY, BRIDGE_KEY, ETHERNET_KEY, INFINIBAND_KEY, WIRELESS_KEY,
```

Comment:
Is the `BRIDGE_KEY` used?

Reply:
not really, had a warning with that but forget to remove it.